php_phongo.c

```diff
@@ -2592,9 +2594,9 @@ static void php_phongo_handshake_data_append(const char* name, size_t name_len,
 	char*  driver_version;
 	char*  full_platform;
 
-	php_version_string_len = strlen(PHP_VERSION);
-	php_version_string     = ecalloc(sizeof(char*), 4 + php_version_string_len);
-	snprintf(php_version_string, 4 + php_version_string_len, "PHP %s", PHP_VERSION);
+	php_version_string_len = strlen(PHP_VERSION) + PHONGO_METADATA_PHP_VERSION_PREFIX_LEN + 1;
+	php_version_string     = ecalloc(sizeof(char*), php_version_string_len);
+	snprintf(php_version_string, php_version_string_len, "%s%s", PHONGO_METADATA_PHP_VERSION_PREFIX, PHP_VERSION);
```

Comment:
I think you're still missing the space that goes after the `PHONGO_METADATA_PHP_VERSION_PREFIX` +  `PHP_VERSION` combo.

Reply:
You are indeed correct. This is an issue in libmongoc which seems to append platform data without a separating space. I'm working on a fix for that, but for the time being this PR will mitigate the issue.